observability/logging/logging-6.0/log6x-clf.adoc

```diff
@@ -5,3 +5,84 @@ include::_attributes/common-attributes.adoc[]
 :context: logging-6x
 
 toc::[]
+
+The `ClusterLogForwarder` CR serves as a single point of configuration for log forwarding, making it easier to manage and maintain log collection and forwarding rules.
+
+* Defines inputs (sources) for log collection
+* Specifies outputs (destinations) for log forwarding
+* Configures filters for log processing
+* Defines pipelines to route logs from inputs to outputs
+* Indicates management state (managed or unmanaged)
+
+// Needs engineering eval re applicability to 6.0
+include::modules/log6x-create-log-forwarder.adoc[leveloffset=+1]
+
+// This needs to be called out extensively.
+.Example ClusterLogForwarder CR
+[source,yaml]
+----
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  name: logging
+  namespace: openshift-logging
+spec:
+  collector:
+    nodeSelector:
+      kubernetes.io/os: linux
```

Comment:
This is not part of the clusterLogForwarder, this is defined in the clusterLogging CR, then, it should be removed from here

@libander